Doc/using/cmdline.rst

```diff
@@ -1036,8 +1037,13 @@ conflict.
 .. envvar:: PYTHONPERFSUPPORT
 
    If this variable is set to a nonzero value, it activates compatibility mode
-   with the ``perf`` profiler so Python calls can be detected by it. See the
-   :ref:`perf_profiling` section for more information.
+   with the ``perf`` profiler so Python calls can be detected by it.
+   This is exactly equivalent to setting :option:`-X perf <-X>`
```

Comment:
It's not equivalent if env vars are ignored. Why not just adding a link to the -X perf option without trying to say that it's *exactly* the same?

Reply:
Note that several other environment variables have a similar description.